src/structures/Superblock.cpp

```diff
@@ -128,6 +128,11 @@ bool Superblock::check_free_inodes() {
     return number_available_inodes > 0;
 }
 
+//Checks whether there are the required number of free blocks. 
+bool Superblock::check_num_free_blocks(int count){
```

Comment:
It's better to rename this func.

Reply:
Like `check_needed_number_of_free_blocks` ?